### DISCUSSION TITLE
AI將成好萊塢化妝師？

### DISCUSSION THREAD

**Discussion by peggy0329** [1101-GS4719-C/Python-Community-GS4719-C#877] [General]
這幾天在IG看到超多人分享美圖秀秀透過AI畫出來的圖片，目前AI辨識技術還有待加強，有些畫出來的圖不免鬧出一些笑話，還蠻有趣的！
但今日看到迪士尼推出的「FRAN」的人工智慧影像技術感覺比近期AI製圖還厲害！可以在維持面部特徵下，讓角色變年輕或老，其技術主要是透過建立由數千張隨機生成面孔的資料庫，透過機器學習方式訓練衍生不同年齡面容，並且再將訓練結果讓「FRAN」進行深度學習，即可讓「FRAN」知曉單一面容在相同場景、相同光影環境，以及相同表情與身體姿勢的不同年齡面容如何呈現。

我認為這樣的技術在幾年後真的**可能使化妝師失業**，畢竟許多電影除了透過化妝師進行基本的妝髮，還是會透過後製進行修飾，如果今天有了FRAN，那其實可以節省化妝師這個人力費用，且妝容效果更佳！

資料來源：https://www.bnext.com.tw/article/73064/disney-fran-faceoff-deage

**Comment by dabbie109303501** [comment]
我有試過傳圖片到美圖秀秀的AI，畫出來的好看程度真的要看運氣...有些還不錯，有些就一言難盡
有了FRAN的技術，感覺要在交友網站上找到照片符合本人的機率就更低ㄌXD

**Reply by peggy0329** [reply]
對ㄟ！這個技術也可以讓大家使用在交友軟體上哈哈哈，到時候大家都照騙哈

**Comment by moonnight3503** [comment]
AI繪圖的功能真的有夠好笑，有時候人還會直接變成背景，看來是還有很多的進步空間呢!

**Reply by peggy0329** [reply]
我朋友直接變成欄杆，超好笑

**Comment by a45320614** [comment]
這個技術太厲害了吧，可以隨意操控臉上的細節，以後不能相信網路上的任何照片了...

**Comment by Chiway1229** [comment]
雖然現在AI繪圖常常會有錯誤跟手指怪怪的問題
不過隨著吃越來越多圖
肯定改善的!

**Comment by dinosaur1009** [comment]
AI製圖感覺還算在草創階段...但感覺以後發展成熟會相當可觀

**Comment by yeh-alex** [comment]
AI繪圖目前主要功能是以繪製草圖為主，如果真的要細圖還是得自己修QQ

**Comment by ching020721** [comment]
如果能應用在影片上，專業化妝師失業的可能性感覺真的會大幅提升

**Comment by StevenWanglolz** [comment]
好酷喔! 看來隨著AI的發展，未來真的會又越來越多職業可以被取代掉耶!

**Comment by Kotori0501** [comment]
我也有試過把我電繪的畫放到美圖秀秀，但AI畫出來的感覺似乎挺怪的，感覺自己的風格都消失的感覺，但我覺得AI繪圖還是可以給我一些靈感來源

**Comment by williamyen0623** [comment]
畫出自己二次元的樣子超酷，有種進入元宇宙的感覺

**Comment by Evan0979** [comment]
試過美圖秀秀的AI繪圖,長的正的帥的畫出來都一樣好看,長的醜的直接變成動物或背景嗚嗚嗚

**Comment by ruthchen1020** [comment]
很酷，但有時候確實很有必要，有時候看30歲的人演高中生也是蠻難受的。

**Comment by evanchou1204** [comment]
感覺隨著這個技術變成熟，網路上的照片或影片都不能相信了，什麼都有可能是AI做的假圖，糟糕。

**Reply by ghost** [reply]
的確

**Comment by CPCpyt** [comment]
我覺得這個技術真的好令人驚豔，也許在未來我們看到的東西都不能稱作真實了。
